Dockerfile

```
@@ -7,8 +7,15 @@ ARG REVISION
 SHELL ["/bin/bash", "-c"]
 
 # Install Node.js
-RUN curl -sL https://deb.nodesource.com/setup_18.x | bash -
-RUN apt-get install -y nodejs
+RUN apt-get update && \
+    apt-get install -y ca-certificates curl gnupg && \
```

PP: `ca-certificates` und `curl` sind bereits installiert.
```suggestion
    apt-get install -y gnupg && \
```